metadata_tools/geometry_support.py

```
@@ -177,40 +191,231 @@ def __init__(self, observation, volume_id, meshgrids, level):
+    def postprocess(self, columns, qualifier):
+        """Process the completed record.
+
+        Args:
+            columns (list): One str for each column. 
+            qualifier (str): 'sky', 'sun', 'ring', or 'body'.
+
+        Returns:
+            None
+        """
+
+        def link_null(link, backplane_keys, data_columns):
```

Ruff (0.14.10)

Missing return type annotation for private function `link_null`

(ANN202)